crud/select/compat/select.lua

```diff
@@ -193,6 +193,7 @@ function select_module.pairs(space_name, user_conditions, opts)
         bucket_id = '?number|cdata',
         force_map_call = '?boolean',
         fields = '?table',
+        fullscan = '?boolean',
```

Comment:
NB: I asked the product team whether we want to issue the warning for `crud.pairs()` (not only `crud.select()`). I'll return back here with an update tomorrow.

Reply:
No pairs mentioned after this comment.
https://github.com/tarantool/tarantool/issues/6539#issuecomment-1073753815

Reply:
@NeraverinTarantool said that the product team decided to don't touch `crud.pairs()`.

It seems, this line (and the similar one in `select_old.lua`) should be removed.

Reply:
The change has been reverted.